smoke-test/turbo-mode/build.gradle

```
@@ -0,0 +1,21 @@
+plugins {
+    id 'java'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'redis.clients:jedis:4.3.2'
+    testImplementation 'org.testcontainers:testcontainers'
+    testImplementation('org.junit.jupiter:junit-jupiter:5.10.0')
+    testImplementation 'ch.qos.logback:logback-classic:1.3.8'
+    testImplementation 'org.assertj:assertj-core:3.24.2'
+}
+
+test {
+    useJUnitPlatform()
+    forkEvery = 1
+    maxParallelForks = Runtime.runtime.availableProcessors().intdiv(2) ?: 1
```

Given the nature of this test, we should probably set it to a fixed value:
```suggestion
    maxParallelForks = 4
```